src/fortran/client.F90

```diff
@@ -408,6 +408,11 @@ end function poll_key
 !> Put a tensor whose Fortran type is the equivalent 'int8' C-type
 function put_tensor_i8(self, name, data, dims) result(code)
   integer(kind=c_int8_t), dimension(..), target, intent(in) :: data !< Data to be sent
+  class(client_type),                    intent(in) :: self !< Fortran SLIC client
```

Comment:
SLIC/SILC is a holdover from the early days of SmarTsim. Maybe rename to "An initialized SmartRedis Fortran client"

Reply:
Sure. Didn't even look at it when I copy/pasted it